test/ModuleInterface/inlinable-function.swift

```diff
@@ -212,7 +212,7 @@ public class HasInlinableDeinit {
   // CHECK: public init(){{$}}
   public init() {}
 
-  // CHECK: @objc @inlinable deinit {
+  // CHECK: [[OBJC:(@objc )?]]@inlinable deinit {
```

Comment:
We should probably just never print `@objc` on `deinit` in textual interfaces. Somehow. But that can happen later; this is clever.